test/README.md

```diff
@@ -3,18 +3,22 @@
 Image Builder needs a database running, the easiest way is to run a container like so:
 
 ```
-sudo podman run -p 5432:5432 --name image-builder-db --health-cmd "pg_isready -u postgres -d imagebuilder"
---health-interval 10s --health-timeout 5s --health-retries 5 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=foobar -e
-POSTGRES_DB=imagebuilder postgres
+sudo podman run -p 5432:5432 --name image-builder-db --health-cmd "pg_isready -u postgres -d imagebuilder" --health-interval 10s --health-timeout 5s --health-retries 5 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=foobar -e POSTGRES_DB=imagebuilder postgres
 ```
 
 Run `make build` and migrate the db (this only needs to be done once) to the latest version:
 
 ```
-PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=foobar PGDATABASE=imagebuilder
-MIGRATIONS_DIR=$HOME/image-builder/internal/db/migrations/ ./image-builder-migrate-db
+PGHOST=localhost PGPORT=5432 PGUSER=postgres PGPASSWORD=foobar PGDATABASE=imagebuilder MIGRATIONS_DIR=$HOME/image-builder/internal/db/migrations-tern ./image-builder-migrate-db-tern
 ```
 
+Open image-builder-migrate-db-tern.go file (under image-builder/cmd folder) and update 2 variables:
```

Comment:
We could probably move this part before "Run `make build`...", what do you think?

Reply:
great idea